lib/pharos/phases/configure_host.rb

```diff
@@ -23,8 +25,31 @@ def call
         logger.info { "Configuring netfilter ..." }
         host_configurer.configure_netfilter
 
-        logger.info { "Configuring container runtime (#{@host.container_runtime}) packages ..." }
-        host_configurer.configure_container_runtime
+        if @host.new?
+          logger.info { "Configuring container runtime (#{@host.container_runtime}) packages ..." }
+          host_configurer.configure_container_runtime
+        else
+          mutex.synchronize {
+            if master_healthy?
+              logger.info { "Draining node ..." }
+              master_ssh.exec!("kubectl drain --force --ignore-daemonsets --delete-local-data #{@host.hostname}")
+            end
+            logger.info { "Reconfiguring container runtime (#{@host.container_runtime}) packages ..." }
+            host_configurer.configure_container_runtime
+            if master_healthy?
+              logger.info { "Uncordoning node ..." }
+              master_ssh.exec!("kubectl uncordon #{@host.hostname}")
+            end
+          }
+        end
+      end
+
+      def master_ssh
+        ssh_manager.client_for(@master)
+      end
+
+      def master_healthy?
```

Comment:
Not sure if this method actually should go to the `Pharos::Configuration::Host`?

Reply:
That requires that hosts know who their master is

Reply:
Or you mean it should be a method for master hosts so you can ask if it thinks it's healthy? Could very well be there, yes.